internal/policy/source/source.go

```diff
@@ -46,6 +46,7 @@ type downloaderFunc interface {
 type PolicySource interface {
 	GetPolicy(ctx context.Context, dest string, showMsg bool) (string, error)
 	PolicyUrl() string
+	PolicyDir() string
```

Comment:
Not sure about this addition, the PolicyDir is a concern of the policy engine, not of the policy source

Reply:
Yeah, that's a reasonable point. It's quite convenient to stash it there though. Anyway, since it's not actually needed yet, I took it out of this PR.